### PR TITLE
Editing my docstrings

### PR DESCRIPTION
**Pull request by hinckleyc**
# Editing my docstrings

## Description

Making my docstrings look more like the docstring for the count_function_without_docstrings function.

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @hinckleyc 

## Reminder

All GitHub Actions should be in a passing state before any pull request is merged.

### CODE REVIEW

**Review by Thompsonn2 — DISMISSED** [review]

File: castanet/counter.py [hunk 1]
```diff
@@ -43,7 +43,14 @@ def count_whileloops(cast_dict):
 
 
 def count_forloops(cast_dict):
-    """A function for counting the number of while loops in a Python program."""
+    """A function for counting the number of while loops in a Python program.
+
+    Args:
+        cast_dict: counts the number of for loops in a file
```
Comment (antlet) [inline castanet/counter.py:49]: The cast_dict argument does not count the number of for loops in a file, rather it is a dictionary of files and their corresponding CAST's. This should be replaced with a statement like that.

File: castanet/counter.py [hunk 2]
```diff
@@ -43,7 +43,14 @@ def count_whileloops(cast_dict):
...
+
+    Returns:
+        int: total number of for loops in a file
```
Comment (antlet) [inline castanet/counter.py:52]: This function returns dictionary not an int. This should be changed to `dict: files and the corresponding for loop amounts` or something similar

File: castanet/counter.py [hunk 3]
```diff
@@ -57,7 +64,14 @@ def count_forloops(cast_dict):
...
 
 def match_if_statements(cast_dict):
-    """A function for counting the number of if statements in a Python program."""
+    """A function for counting the number of if statements in a Python program.
+
+    Args:
+        cast_dict: counts the number of for loops in a file
+
+    Returns:
+        int: total number of while loops in a file
```
Comment (antlet) [inline castanet/counter.py:73]: Refer to previous comments

File: castanet/counter.py [hunk 4]
```diff
@@ -121,8 +135,17 @@ def sum_cast_dict(cast_dict):
     return total
 
 def assignment_count(cast_dict):
-    """A function for counting the number of assignmnet"""
-    # An example of an assignmnet is x = y
+    """A function for counting the number of assignmnet
+
+    Args:
+        cast_dict: counts the number of assignment per file
```
Comment (antlet) [inline castanet/counter.py:141]: Refer to previous comments

File: castanet/counter.py [hunk 5]
```diff
@@ -121,8 +135,17 @@ def sum_cast_dict(cast_dict):
...
+
+    Returns:
+        int: total number of assignments
```
Comment (antlet) [inline castanet/counter.py:144]: Refer to previous comments

File: castanet/counter.py [hunk 6]
```diff
@@ -134,8 +157,17 @@ def assignment_count(cast_dict):
     return assignment_dictionary
 
 def aug_assigment_count(cast_dict):
-    """A function for counting the number of aug assignment"""
-    # An example of an aug assignmnet is x += 5
+    """A function for counting the number of aug assignment
+
+    Args:
+        cast_dict: counts the number of aug assignment per file
```
Comment (antlet) [inline castanet/counter.py:163]: Refer to previous comments

File: castanet/counter.py [hunk 7]
```diff
@@ -134,8 +157,17 @@ def assignment_count(cast_dict):
...
+
+    Returns:
+        int: total number of aug assignments
```
Comment (antlet) [inline castanet/counter.py:166]: Refer to previous comments

**Review by antlet — CHANGES_REQUESTED** [review]
Change the lines where I left comments and it should be all good.

File: castanet/counter.py [hunk 8]
```diff
@@ -121,8 +135,16 @@ def sum_cast_dict(cast_dict):
     return total
 
 def assignment_count(cast_dict):
-    """A function for counting the number of assignmnet"""
-    # An example of an assignmnet is x = y
+    """A function for counting the number of assignmnet
+
+    Args:
+        cast_dict: adds the number of assignment to a dictionary
```
Comment (antlet) [inline castanet/counter.py:141]: This should read `cast_dict: A dictionary of files and the corresponding CAST's`.

File: castanet/counter.py [hunk 9]
```diff
@@ -43,7 +50,14 @@ def count_whileloops(cast_dict):
 
 
 def count_forloops(cast_dict):
-    """A function for counting the number of while loops in a Python program."""
+    """A function for counting the number of for loops in a Python program.
+
+    Args:
+        cast_dict: adds the number of for loops to a dictionary
```
Comment (antlet) [inline castanet/counter.py:56]: This should read `cast_dict: A dictionary of files and the corresponding CAST's`.

File: castanet/counter.py [hunk 10]
```diff
@@ -30,7 +30,14 @@ def match_comment(cast_dict):
 
 
 def count_whileloops(cast_dict):
-    """A function for counting the number of while loops in a Python program."""
+    """A function for counting the number of while loops in a Python program.
+
+    Args:
+        cast_dict: adds the number of while loops to a dictionary
```
Comment (antlet) [inline castanet/counter.py:36]: This should read `cast_dict: A dictionary of files and the corresponding CAST's`.

File: castanet/counter.py [hunk 11]
```diff
@@ -121,8 +135,16 @@ def sum_cast_dict(cast_dict):
     return total
 
 def assignment_count(cast_dict):
-    """A function for counting the number of assignmnet"""
-    # An example of an assignmnet is x = y
+    """A function for counting the number of assignmnet
```
Comment (antlet) [inline castanet/counter.py:138]: This should read `A function for counting the number of assignment statements in a Python program.`

File: castanet/counter.py [hunk 12]
```diff
@@ -134,8 +156,16 @@ def assignment_count(cast_dict):
     return assignment_dictionary
 
 def aug_assigment_count(cast_dict):
-    """A function for counting the number of aug assignment"""
-    # An example of an aug assignmnet is x += 5
+    """A function for counting the number of aug assignment
```
Comment (antlet) [inline castanet/counter.py:159]: This should read `A function for counting the number of aug assignment statements in a Python program.`

**Review by antlet — APPROVED** [review]
Everything looks good now and is ready to merge upon approval.

**Review by Thompsonn2 — DISMISSED** [review]

**Review by MaddyKapfhammer — APPROVED** [review]

**Review by Thompsonn2 — APPROVED** [review]

**Review by antlet — APPROVED** [review]

**Pull request merged**